share/spice/first_robotics/team_info/first_robotics_team_info.js

```diff
@@ -0,0 +1,76 @@
+(function (env) {
+    
+    "use strict";
+    
+    env.ddg_spice_first_robotics_team_info = function(api_result) {
+
+        if (!api_result || api_result.error || api_result.length === 0 || !api_result.name) {
+            return Spice.failed('first_robotics_teams');
+        }
+        
+        // Remove the space for a website if one isn't provided
+        var template_options = {
+            content: 'record',
+            moreAt: true
+        };
+
+        if (api_result.website && api_result.website != null) {
```

Comment:
Though this is fine, you could also use the subtitle. You could either make the team name into the link itself, or you could do something like:

>**FIRST Robotics Competition Team 1114**
>Simbotics | Team Website

You'd need to do something like `subtitle: [ $team_name, { href: $team_website, text: "Team Website" }]`